expression/aggregation/agg_to_pb_test.go

```diff
@@ -19,50 +19,25 @@ import (
 	"fmt"
 	"testing"
 
-	. "github.com/pingcap/check"
-	"github.com/pingcap/parser"
 	"github.com/pingcap/parser/ast"
 	"github.com/pingcap/parser/mysql"
 	"github.com/pingcap/tidb/expression"
-	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/mock"
+	"github.com/stretchr/testify/require"
 )
 
-var _ = Suite(&testEvaluatorSuite{})
-
-func TestT(t *testing.T) {
-	CustomVerboseFlag = true
-	TestingT(t)
-}
-
-type dataGen4Expr2PbTest struct {
-}
-
-func (dg *dataGen4Expr2PbTest) genColumn(tp byte, id int64) *expression.Column {
+func genColumn(tp byte, id int64) *expression.Column {
 	return &expression.Column{
 		RetType: types.NewFieldType(tp),
 		ID:      id,
 		Index:   int(id),
 	}
 }
 
-type testEvaluatorSuite struct {
-	*parser.Parser
-	ctx sessionctx.Context
-}
-
-func (s *testEvaluatorSuite) SetUpSuite(c *C) {
-	s.Parser = parser.New()
-	s.ctx = mock.NewContext()
-}
-
-func (s *testEvaluatorSuite) TearDownSuite(c *C) {
-}
-
-func (s *testEvaluatorSuite) TestAggFunc2Pb(c *C) {
+func TestAggFunc2Pb(t *testing.T) {
+	ctx := mock.NewContext()
```

Comment:
```suggestion
func TestAggFunc2Pb(t *testing.T) {
	t.Parallel()
	ctx := mock.NewContext()
```

I guess this test can be run in parallel. Please confirm, by running locally and check whether it touches some global states.

Reply:
Yes, it works. No error occurs. - 46d7723